google/cloud/bigtable/internal/default_row_reader.cc

```diff
@@ -27,7 +26,7 @@ DefaultRowReader::DefaultRowReader(
     std::string table_name, bigtable::RowSet row_set, std::int64_t rows_limit,
     bigtable::Filter filter, bool reverse,
     std::unique_ptr<bigtable::DataRetryPolicy> retry_policy,
-    std::unique_ptr<BackoffPolicy> backoff_policy)
+    std::unique_ptr<BackoffPolicy> backoff_policy, Sleeper sleeper)
```

Comment:
Does this need to be lambda or can you use a clock?

Reply:
I don't have a strong opinion. LGTm.

Reply:
> I don't have a strong opinion.

Eek. I think you should be strongly in favor of the sleeper.